gdal/port/cpl_vsi_error.cpp

```diff
@@ -61,6 +61,7 @@ typedef struct {
     /* Do not add anything here. szLastErrMsg must be the last field. See CPLRealloc() below */
 } VSIErrorContext;
 
+static void CPL_DLL VSIErrorV(VSIErrorNum, const char *, va_list );
```

Comment:
Thanks, I'm addressing these in a commit; I rebased the travis change so this commit got lost, but I will address the two comments you made on `cpl_vsi_error.cpp`